test/SConscript

```diff
@@ -0,0 +1,64 @@
+import os
+
+Import( 'env' )
+Import( '*' )
+
+if 'loudnessAnalyserLibStatic' in locals() and \
+    'loudnessCorrectorLibStatic' in locals() and \
```

Comment:
Actually it is: https://github.com/mikrosimage/loudness_validator/pull/50/files#diff-25266e934efea80cfcb614c144760f79R50